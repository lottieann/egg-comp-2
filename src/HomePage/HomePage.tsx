import * as React from 'react'

import styled from 'styled-components'

const HomePageContainer = styled.div`
  margin-top: 15vh;
  display: flex;
  justidy-content: center;
  align-items: center;
  flex-direction: column;
`

const StyledTitle = styled.h1`
  @import url('https://fonts.googleapis.com/css2?family=Balsamiq+Sans&display=swap');
  font-family: "Balsamiq Sans", Helvetica;
  font-size: 21px;
  color: black;
`

export const HomePage = () => {
  return (
    <HomePageContainer data-testid="homepage-background">
      <StyledTitle>Welcome to Easter Egg Competition 2021</StyledTitle>
    </HomePageContainer>
  )
}
