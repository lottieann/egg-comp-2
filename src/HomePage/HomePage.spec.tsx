import * as React from 'react'

import 'jest-styled-components'
import '@testing-library/jest-dom/extend-expect'

import { render } from '@testing-library/react'
import { HomePage } from './HomePage'

const setup = () => {
  return <HomePage />
}

describe('HomePage', () => {
  it('should render homepage correctly', () => {
    const { container } = render(setup())

    expect(container).toMatchSnapshot()
  })
})
