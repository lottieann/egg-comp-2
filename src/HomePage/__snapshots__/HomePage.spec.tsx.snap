// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should render homepage correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justidy-content: center;
  -ms-flex-pack: center;
  justidy-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100vh;
  width: 100vw;
  background-image: url('./BreakOut.jpg');
  background-repeat: no-repeat;
  background-position: center center;
  background-size: 100%;
}

.c1 {
  font-family: 'Balsamiq Sans',Helvetica;
  font-size: 21px;
  color: black;
  margin-top: 15vh;
}

<div>
  <div
    class="c0"
    data-testid="homepage-background"
  >
    <h1
      class="c1"
    >
      Welcome to Easter Egg Competition 2021
    </h1>
  </div>
</div>
`;
